src/shared/components/post/post-listing.tsx

```diff
@@ -128,10 +128,10 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
 
   componentDidMount(): void {
     if (UserService.Instance.myUserInfo) {
-      const { auto_expand, blur_nsfw } = UserService.Instance.myUserInfo.local_user_view.local_user;
+      const { auto_expand, blur_nsfw } =
```

Comment:
Hrm, weird how this is getting reformatted. As long as its passing the CI prettier check it should be fine tho.